src/com/walmartlabs/lacinia/pedestal.clj

```diff
@@ -20,6 +20,16 @@
     :headers {}
     :body body}))
 
+
+(defn extract-query-for-type [request]
+  (case (get-in request [:headers "content-type"])
+    "application/json"
```

Comment:
A case could be made here that this should go through a multimethod so that we can easily add support for transit or EDN to upload the query string and variables.